build_info/libaacs-dev.control

```diff
@@ -0,0 +1,19 @@
+Package: libaacs-dev
+Version: @DEB_LIBAACS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Development
+Maintainer: @DEB_MAINTAINER@
+Depends: libaacs0 (= @DEB_LIBAACS_V@) 
```

Comment:
Don't "lock" or restrict dependencies to a specific version.

Reply:
libdevel packages should be locked to the same runtime version

Reply:
Then discard this as resolved. This should only be the case for development packages in the future.